include/kernels/gauge_qcharge.cuh

```diff
@@ -24,16 +24,15 @@ namespace quda
   // Core routine for computing the topological charge from the field strength
   template <int blockSize, typename Float, typename Arg> __global__ void qChargeComputeKernel(Arg arg)
   {
-
     int idx = threadIdx.x + blockIdx.x * blockDim.x;
     int parity = threadIdx.y;
 
     double Q = 0.0;
 
     while (idx < arg.threads) {
       // Load the field-strength tensor from global memory
-      Matrix<complex<Float>, 3> F[6];
-      for (int i = 0; i < 6; ++i) F[i] = arg.data(i, idx, parity);
+      Matrix<complex<Float>, 3> F[] = {arg.data(0, idx, parity), arg.data(1, idx, parity), arg.data(2, idx, parity),
+                                       arg.data(3, idx, parity), arg.data(4, idx, parity), arg.data(2, idx, parity)};
```

Comment:
should the last element be `arg.data(6, idx, parity)` ?

Reply:
Is this a MILC specific optimisation or should it also go into develop too?

Reply:
No matter, I see this is going into develop anyways.

Reply:
And it wasn't an optimization, just a way to silence the compiler warning.